DatadogSessionReplay/Tests/Mocks/SRDataModelsMocks.swift

```diff
@@ -1268,6 +1268,33 @@ extension SRWireframe {
             return nil
         }
     }
+
+    var shapeWireframe: SRShapeWireframe? {
+        switch self {
+        case .shapeWireframe(let value): return value
+        default:
+            XCTFail("not a SRWireframe.shapeWireframe")
```

Comment:
food for thought/ The Mocks shouldn't be linked with a testing platform. For instance, they might be used either in XCTests or SwiftTesting.

Reply:
That's a valid point! Updated ✅